### PR TITLE
Added BETA superscript to page title

### LINKED ISSUE #2515: [MeshMap] Add "BETA" superscript to page title
**Current Behavior**
The [MeshMap beta sign up page](https://layer5.io/service-mesh-management/meshmap) has "MeshMap" as its title.

**Desired Behavior**
Add the word "BETA" as superscript to "MeshMap", like so (except using black font color):

Screen Shot 2022-01-19 at 3 30 48 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by AbhishekJamhoriya**
Signed-off-by: AbhishekJamhoriya jamhoriya.1@iitj.ac.in

Description
Add the word "BETA" as superscript to "MeshMap"
This PR fixes #2515

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1946253b4122e4a483f870e281bcfcbee6d404b8 at: https://61ecbe1083f671ec2d0b43c5--layer5ng.netlify.app

File: src/reusecore/PageHeader/index.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ const PageHeader = ({ category, title, img, feedlink, subtitle,  author, thumbna
         { thumbnail && <div className="feature-image">
           <Image {...thumbnail} imgStyle={{ objectFit: "contain"}} alt={title}/>
         </div>}
-        <h1 className="page-title" >{title} { img && feedlink && (<a href= {feedlink} target="_blank" rel="noreferrer"> <img src={img} alt="RSS Feed"/> </a>) }</h1>
+        <h1 className="page-title" >{title} { img && feedlink && (<a href= {feedlink} target="_blank" rel="noreferrer"> <img src={img} alt="RSS Feed"/> </a>) }
```
Comment (leecalcote) [inline src/reusecore/PageHeader/index.js:51]:
```suggestion
        <h1 className="page-title" >{title} { img && feedlink && (<a href= {feedlink} target="_blank"> <img src={img} alt="RSS Feed"/> </a>) }
```

File: src/sections/Meshmap/meshmap.style.js [hunk 2]
```diff
@@ -2,7 +2,12 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    color: rgb(0, 211, 169);
+    font-size: 75%;
```
Comment (leecalcote) [inline src/sections/Meshmap/meshmap.style.js:7]: Please use `rem` as the unit of measure.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for your contribution, @AbhishekJamhoriya. Double check the font used. It's not being interpreted as Qanelas Soft. Please use the theme styles
Screen Shot 2022-01-22 at 9 56 56 PM
.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by warunicorn19** [comment]
@AbhishekJamhoriya let's discuss this PR on the websites' call.

**Comment by AbhishekJamhoriya** [comment]
@warunicorn19 yes sure.

**Comment by AbhishekJamhoriya** [comment]
Hi @warunicorn19, I am signing off each and every commit then also it's showing a DCO error. why?

**Comment by leecalcote** [comment]
@AbhishekJamhoriya you can see the specific commit missing sign-off here - https://github.com/layer5io/layer5/pull/2536/checks?check_run_id=4917350153 and the instructions on how to remediate this issue, too.

File: src/sections/Meshmap/meshmap.style.js [hunk 3]
```diff
@@ -2,7 +2,11 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    color: #00d3a9;
```
Comment (leecalcote) [inline src/sections/Meshmap/meshmap.style.js:6]: Please make reference to theme color (using variable).

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 08d7c9c9d31be91de765a339bb99f73c6589d148 at: https://61ef5a0bd5a5395b15b740d2--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9aa426d1b5aecc1aa0c402b9c44e4e30fbb656ca at: https://61ef9fc0a38872a350068197--layer5ng.netlify.app

File: src/sections/Meshmap/meshmap.style.js (hunk 3 above)
Comment (AbhishekJamhoriya) [reply src/sections/Meshmap/meshmap.style.js:6]: Lee, what do you mean by using variable.
something like this:
 --white: #ffffff;
color: var(--white)

File: src/sections/Meshmap/meshmap.style.js [hunk 4]
```diff
@@ -2,7 +2,12 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    --theme-color:#00d3a9;
+    color: var(--theme-color);
```
Comment (leecalcote) [inline src/sections/Meshmap/meshmap.style.js:7]: No, this isn't what I mean. I mean use the central theme - https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit c7d7dfab658eebc13f48ac1be359c37c7fc86a8b at: https://61f386135ce6c74a916b2e31--layer5ng.netlify.app

File: src/sections/Meshmap/meshmap.style.js (hunk 4 above)
Comment (Nikhil-Ladha) [reply src/sections/Meshmap/meshmap.style.js:7]: Like `color: ${props => props.theme.black};` here the variable `black` is defined in the themeStyles.js file, which Lee mentioned abovee.

Comment (Nikhil-Ladha) [reply src/sections/Meshmap/meshmap.style.js:7]: But for this case, you need to use `color: ${props => props.theme.caribbeanGreenColor};`

File: src/reusecore/PageHeader/index.js [hunk 5]
```diff
@@ -48,7 +48,9 @@ const PageHeader = ({ category, title, img, feedlink, subtitle,  author, thumbna
         { thumbnail && <div className="feature-image">
           <Image {...thumbnail} imgStyle={{ objectFit: "contain"}} alt={title}/>
         </div>}
-        <h1 className="page-title" >{title} { img && feedlink && (<a href= {feedlink} target="_blank" rel="noreferrer"> <img src={img} alt="RSS Feed"/> </a>) }</h1>
+        <h1 className="page-title" >{title} { img && feedlink && (<a href= {feedlink} target="_blank"> <img src={img} alt="RSS Feed"/> </a>) }
+          <sup className="supscript">{superscript}</sup>
```
Comment (Nikhil-Ladha) [inline src/reusecore/PageHeader/index.js:52]: `{title} {superscript} {RSS Feed Image}` this should be the order.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/sections/Meshmap/meshmap.style.js (hunk 4 above)
Comment (AbhishekJamhoriya) [reply src/sections/Meshmap/meshmap.style.js:7]: ok got it.

File: src/reusecore/PageHeader/index.js (hunk 5 above)
Comment (Nikhil-Ladha) [reply src/reusecore/PageHeader/index.js:52]: This is still not addressed.

**Comment by l5io** [comment]
🚀 Preview for commit c2c2fea3b773c54028330a0462b653289c37ec9d at: https://61f3e765364469b354ca81cb--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
@leecalcote are we good with "Carribena Green" for the `Beta` text or do you think "black" would be better?
Screenshot from 2022-01-28 19-08-53
Screenshot from 2022-01-28 19-09-32

**Comment by l5io** [comment]
🚀 Preview for commit 880213831d33db3c3717cd56a780382c3ea3fe28 at: https://61f413e245ac8a1a3fb9cf9a--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**